contrib/mesos/README.md

```diff
@@ -0,0 +1,46 @@
+# Kubernetes-Mesos
+
+Kubernetes-Mesos modifies Kubernetes to act as an [Apache Mesos](http://mesos.apache.org/) framework.
+
+## Features On Mesos
+
+Kubernetes gains the following benefits when installed on Mesos:
+
+- **Auto-Scaling** - Kubernetes minion nodes are created automatically, up to the size of the provisioned Mesos cluster.
+- **Resource Sharing** - Co-location of Kubernetes with other popular next-generation services on the same cluster (e.g. [Hadoop](https://github.com/mesos/hadoop), [Spark](http://spark.apache.org/), and [Chronos](https://mesos.github.io/chronos/), [Cassandra](http://mesosphere.github.io/cassandra-mesos/), etc.). Resources are allocated to the frameworks based on fairness and can be claimed or passed on depending on framework load.
+- **Infrastructure Independence** - Because Mesos is deployable to nearly every IaaS cloud provider, or even your own data center, Kubernetes-Mesos workloads can easily be migrated from one of these environments to another.
```

Comment:
In fact I just read the global README which also says "runs everywhere". We shouldn't try to compete here with our wordings of "everywhere", at the end irritating our users. This should be really technical and doesn't imply anything:
```
- Mesos can (but of course doesn't have to) run on networks which cannot assign a routable IP to every container. The Kubernetes on Mesos endpoint controller is specially modified to allow pods to communicate with services in such an environment.
```